test/dlc_tests/benchmark/sagemaker/mxnet/training/resources/mx_sm_benchmark.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import os
+
+import boto3
+import sagemaker
+
+from sagemaker.mxnet import MXNet
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--framework-version", type=str, help="framework version in image to be used", required=True)
+parser.add_argument("--image-uri", type=str, help="Image URI of image to benchmark", required=True)
+parser.add_argument("--instance-type", type=str, help="instance type to use for test", required=True)
```

Comment:
Please add the message to `help` that GPU instances are expected to have at least 8 GPUs, because `processes_per_host` has been hard-coded to 8, below.

Reply:
added a comment.